src/core/figures.js

```diff
@@ -10,6 +10,14 @@ import { pub } from "core/pubsubhub";
 export const name = "core/figures";
 
 export function run(conf, doc, cb) {
+  [...doc.querySelectorAll("img")]
```

Comment:
move this to it's own  "function normalizeImages(){}", so it's easier to know what it's doing.